config/config.go

```diff
@@ -0,0 +1,9 @@
+package config
+
+func ImportAPIURL() string {
+	return "http://localhost:21800"
+}
+
+func FilterAPIURL() string {
+	return "http://localhost:22100"
```

Comment:
These values will need to be configurable - see the pattern used in other cmd repos: https://github.com/ONSdigital/dp-observation-importer/blob/cmd-develop/config/config.go